satpy/readers/aapp_l1b.py

```diff
@@ -299,10 +314,11 @@ def calibrate(self,
 
         if dataset_id['name'] in ("3a", "3b") and self._is3b is None:
             # Is it 3a or 3b:
+            line_chunks = get_chunks((self._data.shape[0], 2048))[0]
```

Comment:
as the variable name hints, I looking for the number of lines per chunks. `get_chunks` returns line, col chunks sizes

Reply:
My question was more why are you doing this rather than what are you doing. But I see now that these are scan-based fields, right? So they are one dimensional along-track variables (the row dimension) so there is no need to specify column-wise chunks.

Reply:
yes indeed, sorry for the confusion